refinery/tool_manager/tests.py

```diff
@@ -84,3 +89,69 @@ def test_unallowed_http_verbs(self):
         self.assertEqual(
             self.delete_response.data['detail'], 'Method "DELETE" not '
                                                  'allowed.')
+
+
+class ToolDefinitionGenerationTests(TestCase):
+
+    def test_malformed_workflow_annotation(self):
+        pass
+
+    def test_workflow_improperly_annotated(self):
+        with open("tool_manager/test-data/workflow_bad_annot.json", "r") as f:
+            wf_data = json.loads(f.read())
+            self.assertRaises(
+                CommandError, validate_workflow_annotation, wf_data)
+
+    def test_list_workflow_tool_def_generation(self):
```

Comment:
Okay sure, but I think that considering these tests are ensuring the functionality of the tool definition generation as a whole it would be odd to generate the same ToolDefinition objects N times just for the sake of having one assertion per test method.